tests/foreman/api/test_role.py

```diff
@@ -682,9 +682,7 @@ def test_positive_emptiness_of_filter_taxonomies_on_role_clone(self):
                 None in cloned role
             2. Override flag is set to True in cloned role filter
 
-        :CaseAutomation: notautomated
-
-        :CaseLevel: Integration
+                :CaseLevel: Integration
```

Comment:
It doesn't matter for testimony, but I wonder how and why this get indented.

Reply:
That's easy - because I didn't notice the string has moved when I was deleting staled "notautomated" metadata :-). Fixed.